pkg/geo/geogfn/unary_operators.go

```diff
@@ -90,6 +93,41 @@ func Length(g *geo.Geography, useSphereOrSpheroid UseSphereOrSpheroid) (float64,
 	return length(regions, useSphereOrSpheroid)
 }
 
+// Project returns calculate a projected point given a source point, a distance and a azimuth.
+func Project(point *geom.Point, distance float64, azimuth s1.Angle) (*geom.Point, error) {
+	spheroid := geographiclib.WGS84Spheroid
+
+	// Normalize distance to be positive.
+	if distance < 0.0 {
+		distance = -distance
+		azimuth += math.Pi
+	}
+
+	// Normalize azimuth
+	azimuth = azimuth.Normalized()
+
+	// Check the distance validity.
+	if distance > (math.Pi * spheroid.Radius) {
+		return nil, errors.Newf("distance must not be greater than %f", math.Pi*spheroid.Radius)
+	}
+
+	// Convert to ta geodetic point.
+	x := point.X()
+	y := point.Y()
+
+	projected := spheroid.Project(
+		s2.LatLngFromDegrees(x, y),
+		distance,
+		azimuth)
```

Comment:
trailing comma here too, and on geom.NewPointFlat

Reply:
Done.